chef-extension-windows-vm/azuredeploy.json

```diff
@@ -227,6 +227,12 @@
         "type": "ChefClient",
         "typeHandlerVersion": "1210.12",
         "settings": {
+          "bootstrap_options":
```

Comment:
@NimishaS 
Please update https://github.com/Azure/azure-quickstart-templates/blob/master/chef-extension-windows-vm/azuredeploy-parameters.json for `bootstrap_options`